### PR TITLE
Update the secrets examples to the v1beta3API

### PR DESCRIPTION
**Pull request by mdevilliers**

### CODE REVIEW

**Comment by mdevilliers** [comment]
I signed it!

**Comment by vmarmol** [comment]
Needs a rebase btw

File: docs/secrets.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ This is an example of a simple secret, in json format:
 {
   "apiVersion": "v1beta3",
   "kind": "Secret",
-  "name": "mysecret",
+  "metadata" : {
+  	"name": "mysecret"
+  },
   "namespace": "myns",
```
Comment (bgrant0607) [inline docs/secrets.md]: namespace should also be in metadata.

**Comment by bgrant0607** [comment]
Thank you for the PR. The first secret does still need to be fixed. It was missed in PR #8363. Please rebase so that we can merge the remaining necessary fix.

@caesarxuchao Please review the whole file, not just the change.

cc  @nikhiljindal @pmorie

**Comment by bgrant0607** [comment]
LGTM

**Comment by caesarxuchao** [comment]
There is an extra "," after namespace. I will submit another PR, as the json for the pod is also broken. I should have caught these in #8363 ...

**Pull request merged**